### PR TITLE
Test fixes cleanup

### PR DESCRIPTION
**Pull request by kurund**
## Summary

* Clean up
* Test fixes for my account
* Test fixes for Groups
* Test fixes for GroupContacts
* Test fixes for GroupContactList
* Speed Send test fixes

## Test Plan

* Included in the PR

### CODE REVIEW

File: src/containers/Group/Group.test.tsx [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
-            description: 'Only for staff members',
-          },
-        },
-      ],
-    },
-  },
+  getUsersQuery,
+  ...getOrganizationQuery,
+  getGroupQuery,
+  getGroupQuery, // if you refetch then you need to include same mock twice, learnt it hard way
```
Comment (rathorevaibhav) [inline src/containers/Group/Group.test.tsx:15]: 🙂 
```suggestion
  getGroupQuery, // if you refetch then you need to include same mock twice
```

File: src/containers/Group/Group.test.tsx [hunk 2]
```diff
@@ -1,33 +1,45 @@
...
+    expect(getByText('Edit group')).toBeInTheDocument();
+
+    // remove first user
+    const removeUser = getAllByTestId('deleteIcon');
+    UserEvent.click(removeUser[0]);
+
+    // click on SAVE
+    const saveButton = screen.getByText('Save');
+    UserEvent.click(saveButton);
+    await wait();
```
Comment (rathorevaibhav) [inline src/containers/Group/Group.test.tsx:43]: should we write an expect statement after this?

File: src/containers/Template/Form/SpeedSend/SpeedSend.test.tsx [hunk 3]
```diff
@@ -52,15 +52,21 @@ describe('SpeedSend', () => {
...
     const button = queryByText('Save');
     fireEvent.click(button);
     await wait();
     await wait();
-    const { getByText } = within(container.querySelector('tbody'));
-    expect(getByText('Good message')).toBeInTheDocument();
+
+    expect(queryByText('Message is required.')).toBeInTheDocument();
+
+    // const { getByText } = within(container.querySelector('tbody'));
```
Comment (rathorevaibhav) [inline src/containers/Template/Form/SpeedSend/SpeedSend.test.tsx:69]: should we remove this if not needed?

File: src/containers/MyAccount/MyAccount.tsx [hunk 4]
```diff
@@ -65,7 +65,10 @@ export const MyAccount: React.SFC<MyAccountProps> = () => {
         setShowOTPButton(false);
       })
       .catch((error) => {
-        console.log(error);
+        setToastMessageInfo({
+          severity: 'error',
+          message: `Enable to send an OTP to ${loggedInUserPhone}.`,
```
Comment (rathorevaibhav) [inline src/containers/MyAccount/MyAccount.tsx:70]:
```suggestion
          message: `Unable to send an OTP to ${loggedInUserPhone}.`,
```

File: src/containers/Group/GroupList/GroupList.test.tsx [hunk 5]
```diff
@@ -1,14 +1,31 @@
...
+  test('should render GroupList', async () => {
+    const { getByText } = render(wrapper);
+
+    // loading is show initially
+    expect(getByText('Loading...')).toBeInTheDocument();
+
+    await wait();
+    expect(getByText('Groups')).toBeInTheDocument();
+
+    // test automation
```
Comment (rathorevaibhav) [inline src/containers/Group/GroupList/GroupList.test.tsx:27]:
```suggestion
    // TODO: test automation
```

File: src/containers/Group/GroupList/GroupList.test.tsx [hunk 6]
```diff
@@ -1,14 +1,31 @@
...
+
+    // test delete
```
Comment (rathorevaibhav) [inline src/containers/Group/GroupList/GroupList.test.tsx:29]:
```suggestion
    // TODO: test delete
```

File: src/containers/MyAccount/MyAccount.test.helper.ts [hunk 7]
```diff
@@ -1,3 +1,4 @@
+import { UPDATE_CURRENT_USER } from '../../graphql/mutations/User';
 import { GET_CURRENT_USER } from '../../graphql/queries/User';
 
 export const MY_ACCOUNT_MOCKS = [
```
Comment (rathorevaibhav) [inline src/containers/MyAccount/MyAccount.test.helper.ts:4]: should we move this file to the `Mocks/` directory?

File: src/containers/Group/Group.test.tsx (hunk 2 above)
Comment (kurund) [reply src/containers/Group/Group.test.tsx:43]: Done

File: src/containers/Template/Form/SpeedSend/SpeedSend.test.tsx (hunk 3 above)
Comment (kurund) [reply src/containers/Template/Form/SpeedSend/SpeedSend.test.tsx:69]: We should revisit this once we add TODO on line 55.

File: src/containers/MyAccount/MyAccount.test.helper.ts (hunk 7 above)
Comment (kurund) [reply src/containers/MyAccount/MyAccount.test.helper.ts]: Done

**Review by rathorevaibhav — APPROVED** [review]

**Pull request merged**